### PR TITLE
fix/Estimations announcement

### PR DESCRIPTION
**Pull request by carpawell**

### CODE REVIEW

**Comment by carpawell** [comment]
Seems like we also need a routine that is going to iterate over containers and be ready to clean up every info on every container removal (also, the storage engine should support this operation too, of course). But it is a too big change so not added it yet to this PR. We have some [removal subscription](https://github.com/nspcc-dev/neofs-node/blob/4bbe8f99e30e4fde727e9c3ea36e695ed58e74cf/cmd/neofs-node/container.go#L109) but it won't work for the removals at the SN's offline time.

Thoughts? @roman-khimov, @cthulhu-rider, @AliceInHunterland

**Comment by roman-khimov** [comment]
> clean up every info on every container removal

Are you talking about #1663 or some kind of locally stored metabase thing related to estimations specifically?

**Review by roman-khimov — APPROVED** [review]

**Comment by carpawell** [comment]
> some kind of locally stored metabase thing related to estimations specifically?

Like this, yes. Iff a node is sleeping it won't be able to drop its indexes with the notifications approach (that we already have). But yes, now it does _absolutely nothing_ if we are talking about the estimations but I do not want to fix it fast cause notifications it the only part of potentially useless info in the metabase if a container is dropped, so want to hear your thoughts and maybe create another issue for more accurate solution.

But the current missing container issue is fixed with this PR, although I would not treat it as the best solution for _the exact that_ problem.

**Comment by roman-khimov** [comment]
I'd forget this problem for now, estimation mechanism itself won't be like this forever.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Changelog for NeoFS Node
 ### Fixed
 - `neofs-cli netmap netinfo` documentation (#2555)
 - `GETRANGEHASH` to a node without an object produced `GETRANGE` or `GET` requests (#2541)
+- Iteration over locally collected container estimation values (#)
```
Comment (roman-khimov) [inline CHANGELOG.md:9]: Number?

Comment (carpawell) [reply CHANGELOG.md]: fixed

**Review by cthulhu-rider — APPROVED** [review]

**Comment by roman-khimov** [comment]
Needs a rebase, CHANGELOG has changed.

**Pull request merged**